src/solidity-frontend/solidity_grammar.cpp

```diff
@@ -65,21 +85,19 @@ TypeNameT get_type_name_t(const nlohmann::json &type_name)
   if(type_name.contains("typeString"))
   {
     // for AST node that contains ["typeName"]["typeDescriptions"]
-    if(type_name["typeString"] == "uint8" || type_name["typeString"] == "bool")
+    std::string typeString = type_name["typeString"].get<std::string>();
+
+    if(uintTypeStringSet.count(typeString) || typeString == "bool")
```

Comment:
It seems that this is the only place `uintTypeStringSet` is used. Basically for checking whether the `typeString` matches one of these constants. Wouldn't it be possible to use `uintStringToTypeMap` for that purpose, since its keys are the same as in the set? Then the set could be removed.